### ISSUE TITLE
Mesh view is washed out in visualizer, texture not embedded

### ISSUE THREAD

**Issue by Mikerhinos** [naver/dust3r#3]
Fantastic work guys ! Quality and performance is impressive, though the visualizer seems to apply some transparency in mesh mode compared to point cloud.

Here is a comparaison of the 2 using a single image as source : 

![image](https://github.com/naver/dust3r/assets/18692487/44347e32-4d43-4c14-b443-f00434e52775)
![image](https://github.com/naver/dust3r/assets/18692487/9373b00d-26a6-4783-9c8a-66dc112890b8)

Also when saving the mesh there is no texture embedded and no color information on the point cloud when I import it to Blender

![image](https://github.com/naver/dust3r/assets/18692487/a7ba97db-189d-43dc-b034-72e14635cd8e)

PS : I installed dust3r using Pinokio.

**Comment by vincent-leroy** [comment]
Yes we also noticed the "washed-out" effect on the meshes, but unfortunately I don't think we can change this, unless Gradio updates it's mesh viewer...
Regarding import to blender, this video does import colored meshes https://www.youtube.com/watch?v=vY7GcbOsC-U , I hope this helps !

**Comment by Mikerhinos** [comment]
Yes you're right it seems to be a Model3D Gradio component problem with lights https://github.com/gradio-app/gradio/issues/5430 someone apparently fixed it but for some reason deleted the Huggingface space :/

For Blender color display it's working fine thanks to the video, for any people wanting to quickly display it just go to the shading drop down menu while in solid color view and in the color section change from "Material" to "Attribute".
